app/src/lib/dispatcher/sign-in-store.ts

```diff
@@ -23,6 +23,8 @@ function getUnverifiedUserErrorMessage(login: string): string {
   return `Unable to authenticate. The account ${login} is lacking a verified email address. Please sign in to GitHub.com, confirm your email address in the Emails section under Personal settings, and try again.`
 }
 
+const EnterpriseTooOldMessage = `The GitHub Enterprise version is too old. Talk to your server's administrator about upgrading to the latest version of GitHub Enterprise.`
```

Comment:
First sentence tweak: `The GitHub Enterprise version does not support GitHub Desktop.`